src/firefly/java/edu/caltech/ipac/firefly/server/dpanalyze/SofiaAnalyzer.java

```diff
@@ -44,6 +39,9 @@ public FileAnalysisReport analyzeFits(FileAnalysisReport inputReport,
         String code = params.getOrDefault("product_type", "");
         String level = params.getOrDefault("processing_level", "").toUpperCase();
         String inst = params.getOrDefault("instrument", "").toUpperCase();
+        Map<String, String> additionalParams = new HashMap<>();
+        additionalParams.put("CUBE_FIRST_FRAME","50%");
+        inputReport.setAdditionalImageParams(additionalParams);
```

Comment:
The code is fine but since you are just setting one parameter  you could just use a singleton map.

```java
inputReport.setAdditionalImageParams(Collections.singletonMap("CUBE_FIRST_FRAME","50%"));
```